kubernetes/client/api/__init__.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+
+# flake8: noqa
+
+# import apis into api package
```

Comment:
This packages is renamed from apis to api.

Reply:
Out of curiosity, why did you change it from apis to api?